archetypes/author.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "" # Name
```

Comment:
Excellent comment here, I certainly would have been confused about what title meant. Could we update this to `# Your full name`">, please?

Reply:
Sure